### DISCUSSION TITLE
Using Heic, I have an error that I don't understant

### DISCUSSION THREAD

**Discussion by lasserrearnaud** [gotson/NightMonkeys#63] [Q&A] [answered]
Hi, first of all, thank you for your project, it's exactly what I need.
I have installed libheic, configure my java project to fit your requirements, but when i run ImageIO.read I face an HeifException: Invalid input: No 'meta' box. If i try by passing my heic image in the reader, I have No 'ftyp' box.
What does box mean ?
Did you already had this kind of error ?
I've tried many images to be sure that what not the problem.

**Comment by gotson** [comment]
i haven't experienced this issue yet. Would you be able to share the image, i can run it in the test harness and see if it works or not.

**Reply by lasserrearnaud** [reply]
I've use the pics from the test folder of your project, also from my phone... I don't think it comes from the images, is there an other lib(than libheif) to download (i'm on Ubuntu) to make the pluggin works ?

**Reply by gotson** [reply]
normally you only need libheif v1.16+

**Reply by gotson** [reply]
i found [this](https://github.com/strukturag/libheif/issues/595) while searching for your error message. Maybe it's because you have an old version of libheif ?

**Reply by lasserrearnaud** [reply, marked as answer]
Hi, using the v1.0.0 with java22 resolved my problem, thank you for your work !
